azure-pipelines.yml

```diff
@@ -289,8 +304,15 @@ jobs:
       verilator --version
       verible-verilog-lint --version
     displayName: Display environment
-  - bash: ci/scripts/build-chip-verilator.sh earlgrey
-    displayName: Build simulation with Verilator
+  - bash: |
+      ./bazelisk.sh build //hw:verilator
```

Comment:
Ugh, I forgot to do this the portable way for buildifier and in quick-bazel.sh. This is definitely the right way to do it.